packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -670,17 +671,18 @@ export class BiconomySmartAccountV2 extends BaseSmartContractAccount {
   async sendUserOp(userOp: Partial<UserOperationStruct>, params?: SendUserOpParams): Promise<UserOpResponse> {
     delete userOp.signature;
     const userOperation = await this.signUserOp(userOp, params);
-    const bundlerResponse = await this.sendSignedUserOp(userOperation);
+    const bundlerResponse = await this.sendSignedUserOp(userOperation, params?.simulationType);
     return bundlerResponse;
   }
 
   /**
    *
-   * @param userOp
+   * @param userOp - The signed user operation to send
+   * @param simulationType - The type of simulation to perform ("validation" | "validation_and_execution")
    * @description This function call will take 'signedUserOp' as input and send it to the bundler
    * @returns
    */
-  async sendSignedUserOp(userOp: UserOperationStruct): Promise<UserOpResponse> {
+  async sendSignedUserOp(userOp: UserOperationStruct, simulationType?: SimulationType): Promise<UserOpResponse> {
```

Comment:
this would probably conflict with params once we add state overrides stuff. it's probably better to keep separate type SendSignedUserOpParams which is passed a second argument.

Reply:
once we have that object, in another PR we can add overrides in same dto.

so simulationType is only relevant for biconomy bundler
but stateOverrideSet may be accepted by one or more bundlers 
(will have to mention this in comments)

Reply:
Why would it conflict ? state overrides is a param that will be passed to buildUseropDto, as it can be used only on estimate user op gas so that param will not be on sendSignedUserOp